docs/sql/mysql/create_schema.sql

```diff
@@ -148,6 +148,7 @@ CREATE TABLE `pa_activation_history` (
   `event_reason` varchar(255),
   `external_user_id` varchar(255),
   `timestamp_created` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `version` int(2) NOT NULL,
```

Comment:
Očekával bych nějaký _constraint_, ale nenašel jsem ho ani v tabulce `pa_activation`.

Reply:
We do not have constraints for PA protocol versions at the moment. Perhaps an idea for improvement, but we will not add it as part of this pull request.